### ISSUE TITLE
curator fails to add study in treebase

### ISSUE THREAD

**Issue by plewis** [OpenTreeOfLife/opentree#641]
Tried to add TreeBase study ID 14763 at https://tree.opentreeoflife.org/curator/study/create, but received "Sorry, there was an error creating this study. (No more information is available.)" The problem may result from the fact that there are errors (e.g. misspelled journal name) and missing information (no volume or page range) in the citation: 

Fucikova K., Lewis P., & Lewis L. 2014. Widespread desert affiliation of trebouxiophycean algae (Trebouxiophyceae, Chlorophyta) including discovery of three new desert genera. Phycological Reserach

**Comment by mtholder** [comment]
Thanks for the report @plewis 
The study ID should bypass any naming issues with the citation. I can reproduce this, so we should be able to look into this quickly.

@jimallman  you might have some thoughts, when I reproduce what Paul did i get the same error. If I turn on the developer tools in Chrome, I see:

importMethod: [import-method-TREEBASE_ID]
jquery.js:2 POST https://api.opentreeoflife.org/phylesystem/v1/study 500 (INTERNAL SERVER ERROR)send 
@ jquery.js:2v.extend.ajax 
@ jquery.js:2createStudyFromForm 
@ study-creation.js:230(anonymous function)
@ study-creation.js:185v.event.dispatch
@ jquery.js:2o.handle.u
@ jquery.js:2

And I get a payload of {"description": "study ID expected after \"study/\"", "error": 1} from that URL.

I'm afraid that I don't recall the API off the top of my head. Is the study-creation.js line 230 in the curator app supposed to be using v2, perhaps?

**Comment by mtholder** [comment]
Hi @plewis . I finally found a few minutes to look into this.
A fix is not deployed. The problem appears to be an empty tree in the tree base record:

It looks like TreeBASE is not returning the tree for 18S.
See both the NEXUS and NeXML download from http://treebase.org/treebase-web/search/study/summary.html?id=14763

In NeXML, the tree ID is Tr75035. It emits a tree with an unlabeled node and no edges. My script was expecting at least one edge.  I'll submit pull requests to get my fixes deployed, but as it stands I'm not sure what to do with the empty tree. I'm not sure if it will cause problems elsewhere in our pipeline.

The problem with the TB record could be a problem with the upload, or TreeBASE. I did report this at:
   https://sourceforge.net/p/treebase/bugs/250/
in case it is a software issue.
